src/temaWeek7HashMap/Persoane.java

```diff
@@ -13,7 +13,6 @@
 //		USE THE ALREADY DEFINED CLASSES: PERSOANA, SOMER, ANGAJAT, STUDENT; ADRESA; TARA;
 
 
-
 import java.util.*;
 
 public class Persoane  {
```

Comment:
Same remark about the main() method.
I enjoyed the way you iterated over the map. Thumbs up for using the forEach() method, which takes a Consumer as parameter (passed in as a lambda expression).